README.md

```diff
@@ -105,6 +105,14 @@ source /FULL/PATH/TO/wp-completion.bash
 
 Don't forget to run `source ~/.bash_profile` afterwards.
 
+If using zsh for your shell, you may need to do the following:
```

Comment:
> does that not agree with my instructions here?

I don't think it's communicated in "you may need to do the following".

It would be better as: "you should add the following to your `~/.zsh_profile`"